src/rez/rezconfig.py

```diff
@@ -948,6 +948,9 @@
 #       launched without extension from windows and other systems.
 create_executable_script_mode = "single"
 
+# Default build process to use during build/release
```

Comment:
Should the valid values be documented? We could also make a link to `--process`.

Reply:
I've added a small comment on the build processes currently available.